ivy_tests/test_ivy/test_exceptions.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+
+import ivy
+
+
```

Comment:
the tests shouldn't simply check that the modes are set correctly, they should also raise exceptions and verify that the stack traces are what is expected. Basically you should create ground truth stack trace strings and assert that the stack traces produced are what we expect. This will also make it easy for me to verify that things are working as expected :)

Reply:
Oh okay, I see. I'll do that.

Reply:
Hey! Probably because I'm the assignee so I got the related notifications aha

Anyways, I have an idea / suggestion, will it work if we test that certain keywords of the stack trace is in the traces (and also test fails if keywords that are not supposed to be there is included in the traces)

for example, if the mode is `frontend`, keywords like "backends", "site-packages" (which refer to the actual backend framework library, because asserting "torch" or "jax" for example, may also refer to frontends which can be acceptable in some cases) will cause the test to fails because they belongs to the backend traces and should not appear in the stack trace string

just a basic idea aha, hope this helps and feel free to modify or improve based on this, or implement a better one if you now have one! 

Thanks!

Reply:
Hey @juliagsy yeah I think this could work! Thanks! Although, we won't have ground truths anymore which I believe would have been a sure way for @djl11 to know that things work exactly as expected. But I do think your suggestion is better than no tests at all!